draft-ietf-httpbis-digest-headers.md

```diff
@@ -253,9 +254,11 @@ STRUCTURED-FIELDS}}):
 Representation-Digest   = sf-dictionary
 ~~~
 
-`Representation-Digest` member-keys convey the hashing algorithm (see
-{{algorithms}}) used to compute the digest. Member-values are the output
-of the digest calculation. Member-values MUST be of type sf-binary.
+where member-keys convey the hashing algorithm (see
+{{algorithms}}) used to compute the digest
+and have no parameters.
+Member-values are the output of the digest calculation
+and their syntax is `sf-binary`.
```

Comment:
How about we just move this to above the ABNF. So it reads

> `Representation-Digest` is a Structured Fields Dictionary (see {{Section 3.2 of
STRUCTURED-FIELDS}}). Member-keys convey ....
> 
> ~~~ abnf